libnavigation-core/src/main/java/com/mapbox/navigation/core/sensor/SensorData.kt

```diff
@@ -0,0 +1,147 @@
+package com.mapbox.navigation.core.sensor
+
+import android.os.SystemClock
+import com.mapbox.bindgen.Value
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigator.SensorType
+
+/**
+ * Data obtained from sensors
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+sealed class SensorData {
```

Comment:
It's not good if we want to extend it in the future. Since it's experimental, we can break it, but it might happen that we leave it as is when we remove experimental annotation. I'd suggest removing the sealed keyword now.